stories/clickable_link.fmf

```diff
@@ -0,0 +1,45 @@
+/clickable_test_link
+    story:
+        As a tester I want to link test verifying Jira issue.
+
+    description:
+        User can click on a link in JIRA issue that returns human-readable metadata of test that covers
+        this issue.
```

Comment:
Let's limit this to linking only (clicking part to be covered by the human-readable story). Perhaps something like this?

    For cases when there is a single test verifying given issue
    there would be fmf identifier pointing to that individual
    test.